src/lib/support/jsontlv/TlvToJson.cpp

```diff
@@ -173,11 +171,8 @@ CHIP_ERROR TlvStructToJson(TLV::TLVReader & reader, Json::Value & jsonObj)
         TLV::Tag tag = reader.GetTag();
         VerifyOrReturnError(TLV::IsContextTag(tag) || TLV::IsProfileTag(tag), CHIP_ERROR_INVALID_TLV_TAG);
 
-        // Profile tags are expected to be implicit profile tags and they are
-        // used to encode > 8bit values from json
-        if (TLV::IsProfileTag(tag))
+        if (TLV::IsProfileTag(tag) && (TLV::VendorIdFromTag(tag) == 0))
```

Comment:
Why the extra parens?  And inconsistently?

Reply:
Removded